presto-docs/src/main/sphinx/security/file-based-system-access-control.rst

```diff
@@ -2,46 +2,47 @@
-file. For example, if a JSON file named ``rules.json`` resides in ``etc``,
-add an ``etc/access-control.properties`` with the following contents:
+A system access control plugin enforces authorization at a global level,
+before any connector level authorization. You can either use Presto's built-in
+plugin or provide your own by following the guidelines in
+:doc:`/develop/system-access-control`.
+
+The built-in file based system access control plugin allows you to specify
+access control rules in a file. To use this plugin, add an
+``etc/access-control.properties`` file containing two required properties:
+``access-control.name``, which must be equal to ``file``, and
+``security.config-file``, which must be equal to the location of the config file.
+For example, if a config file named ``rules.json``
+resides in ``etc``, add an ``etc/access-control.properties`` with the following
+contents:
 
 .. code-block:: none
 
    access-control.name=file
    security.config-file=etc/rules.json
 
-The ``rules.json`` file should consist of a list of access control rules.
-These rules should be specified in the order in which they are intended to be
-matched. The user is granted the privileges from the first matching rule. All
+The config file consists of a list of access control rules in JSON format. The
+rules are matched in the order specified in the file. All
 regular expressions default to ``.*`` if not specified.
 
-At present, only the catalog access control rules are supported in this
-plugin. Other access control rules such as those for schemas or tables or
-session properties are not yet supported.
+This plugin currently only supports catalog access control rules. If you want
+to limit access on a system level in any other way, you must implement a custom
+SystemAccessControl plugin (see :doc:`/develop/system-access-control`).
 
 Catalog Rules
 -------------
 
-These rules govern the accesses allowed on specified catalogs for the specified
-users. The user is granted the access to a catalog based on the first matching
-rule. In case no rule matches, the user is denied the access to the catalog.
-Thus, in absence of any rules, access to all catalogs is denied for all users
-when this plugin is enabled. Each rule is composed of the following fields:
+These rules govern which catalogs particular users can access. The user is
+granted access to a catalog based on the first matching rule. If no rule
+matches, access is denied. Each rule is composed of the following fields:
 
 * ``user`` (optional): regex to match against user name.
-
 * ``catalog`` (optional): regex to match against catalog name.
-
 * ``allowed`` (required): boolean indicating whether a user has access to the catalog
 
-Consider a use case in which the only access control restriction you want is
-to disallow access to ``mysql`` catalog to user ``alice``. This can be achieved
-by the following rules:
+For example, if you want to prohibit the user ``alice`` from accessing
```

Comment:
like I said before, can you make this example that you
* prohibit alice from access mysql
* allow all users starting with the letter a (e.g. a.*) to access all catalogs
* Deny access for all other users